Embedded-Kernel/frontend
# Register and Sign up feature

#### Description of changes made
---

> Sign in modal and sign up modal

#### Type of change
---

> **Please select the relevant options**


#### How Has This Been Tested?
---

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?
---

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/takitrw/payingtone-app), and `cd [the project name]`
- Check out this branch by `git checkout register`
- Run `yarn add` to install the project dependencies
- Run `yarn run dev` for starting.
- Check `localhost:3000` in your browser.
- Click the Login button in the top navigation bar.
- Boom you will see a login modal. 
- If you don't have an account yet click the sign up page.
- Have a look in your mobile emulator on the sign-in screen.
- .....

#### Checklist:



#### Screenshots
 

<img width="917" alt="signup" src="https://user-images.githubusercontent.com/73523419/138824036-eccbf6a2-ef52-4a72-86c1-9a40565eb2a6.PNG">

- [ ] Sign up

<img width="926" alt="loginPage" src="https://user-images.githubusercontent.com/73523419/138823946-c807ff16-841f-499a-9b64-f591c803a8e8.PNG">
ers

### Review


- @veritem 
-  @nshimiyeemmy 
- @chanelle740 
- @SauveJeanLuc
